src/Symfony/Component/Console/Application.php

```diff
@@ -762,7 +762,7 @@ protected function doRenderException(\Exception $e, OutputInterface $output)
             }
 
             $messages = array();
-            if (!$e instanceof ExceptionInterface || OutputInterface::VERBOSITY_VERBOSE <= $output->getVerbosity()) {
+            if (!$e instanceof ExceptionInterface && OutputInterface::VERBOSITY_VERBOSE > $output->getVerbosity()) {
```

Comment:
the superior to condition seems weird, isn't?

Reply:
reverted